tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/BoundaryCorrections.hpp

```diff
@@ -44,6 +44,45 @@ struct inverse_spatial_metric_tag<true> {
   template <typename System>
   using f = typename System::inverse_spatial_metric_tag;
 };
+
+template <size_t Dim>
+void adjust_inverse_spatial_metric(
+    const gsl::not_null<tnsr::II<DataVector, Dim>*> inv_spatial_metric) {
+  // Make sure the inverse spatial metric is delta_{ij}+something smaller
+  // so that it's physically reasonable.
+  for (size_t i = 0; i < Dim; ++i) {
+    for (size_t j = i; j < Dim; ++j) {
+      inv_spatial_metric->get(i, j) *= 0.01;
+    }
+  }
+  for (size_t i = 0; i < Dim; ++i) {
+    inv_spatial_metric->get(i, i) += 1.0;
+  }
+}
+
+template <size_t Dim>
+void normalize_vector_and_covector(
```

Comment:
Here also, can you document which of these not nulls are input, which are output, and what happens? (I don't expect super detailed dox, but just a sentence to help the reader. Perhaps: "Compute the normal vector from the normal covector and the inverse spatial metric, then scale the vector and covector so they have unit magnitude" or similar)

This is when I miss Fortran's `in`, `out`, `inout` intent specifications, which are so so helpful in interpreting arguments 😛